### PR TITLE
Add ability to set custom lambda handler

### PR DESCRIPTION
**Pull request by rulio**
Related to #4

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -14,12 +14,13 @@ services:
...
       - SHIP_NAME=docker-ship
+      - HEADQUARTERS_REMOTE=git@github.com:glg/ec2.starphleet.dev.headquarters.git#RaulReynosoDevAcc
       - SERVERLESS_TARGET_AWS_ACCOUNT=dev
       - SERVERLESS_SECURITY_MODE=public    
       - SERVERLESS_MEMORY_SIZE=2048
       - SERVERLESS_TIMEOUT=20
       - SERVERLESS_RUNTIME=nodejs6.10  
       - SERVERLESS_UPDATE_IN_PLACE=true
-      - HEADQUARTERS_REMOTE=git@github.com:glg/ec2.starphleet.dev.headquarters.git#RaulReynosoDevAcc
+      # - SERVERLESS_LAMBDA_HANDLER=my_own_lambda_handler.api # Optionally set a custom handler.
```
Comment (fearphage) [inline docker-compose.yml]: Did you mean to leave this comment in?

File: test/services/apps/whoami/my_own_lambda_handler.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+const serverless = require('serverless-http');
+const app = require('./app');
+
+module.exports.api = serverless(app);
```
Comment (fearphage) [inline test/services/apps/whoami/my_own_lambda_handler.js]: I see this test here, but how do you run the tests?

File: README.md [hunk 3]
```diff
@@ -105,6 +105,24 @@ Determines whether deployment will create a new API Gateway and Lambda Function
 - Default: `false`
 - Valid Values: `true`,`false`
 
+### SERVERLESS_LAMBDA_HANDLER
+Optionally sets the handler that the lambda function will use.
```
Comment (fearphage) [inline README.md]: There are [other instructions](https://github.com/glg/serverless-buildpack#step-1---create-compatible-node-service) that say the file name is required: 

>  MUST NOT include a file named `lambda_index.js` (Serverless Buildpack adds a file with this name)

This should be update to say this defaults to X, but you can change it with Y.

File: test/services/apps/whoami/my_own_lambda_handler.js (hunk 2 above)
Comment (rulio) [reply test/services/apps/whoami/my_own_lambda_handler.js]: It's a manual test.  This is a test app to deploy to serverless from your local machine. You would do:
```
docker-compose up -d
docker-compose exec  buildpack ./bin/compile /var/buildpack/services/apps/whoami/
```

The second command would deploy the app to serverless in AWS.  You would have to set your some AWS env vars that get passed into docker container via compose yaml.

File: docker-compose.yml (hunk 1 above)
Comment (rulio) [reply docker-compose.yml]: No, i just left it there for reference.  But didn't want to use it by default the settings here are used when deploying the test app.

**Pull request merged**